cg/server/api.py

```diff
@@ -356,6 +366,23 @@ def parse_flow_cell(flowcell_id):
     return jsonify(**flow_cell.to_dict(samples=True))
 
 
+@BLUEPRINT.route("/flowcells/<flow_cell_id>/sequencing_metrics", methods=["GET"])
+def get_sequencing_metrics(flow_cell_id: str):
+    """Return sequencing metrics for a flow cell."""
+
+    if not flow_cell_id:
+        return jsonify({"error": "Invalid or missing flow cell id"}), http.HTTPStatus.BAD_REQUEST
+
+    metrics: List[
```

Comment:
I think metrics is a bit too general for variable names
```suggestion
    sequencing_metrics: List[
```

Reply:
https://github.com/Clinical-Genomics/cg/pull/2268/commits/a7d6e1c8702948b58780099269fd52b62c258c22